src/install.ts

```diff
@@ -21,8 +21,9 @@ export async function activate(
       'clangd.install', async () => common.installLatest(ui)));
   context.subscriptions.push(vscode.commands.registerCommand(
       'clangd.update', async () => common.checkUpdates(true, ui)));
-  const status = await common.prepare(ui, config.get<boolean>('checkUpdates'));
-  return status.clangdPath;
+  const status =
```

Comment:
The function comment says "returns the path or null", so I think we return status?.clangdPath instead maybe, and change the signature?

Reply:
Good catch, the caller of this function already checks if the result is falsy. I'll fix that and add `null` to the return type.